src/science/casa-cnp/casa_rplant.F90

```diff
@@ -1,3 +1,4 @@
+!#define ESM15 YES
```

Comment:
Actually, the intent is to remove ESM15 here. So let the `#define` out. And remove the `#ifdef` cases. They are not needed anymore since `casaflux%frac_sapwood` is initialised to 1.